packages/meditrak-server/src/routes/accessRequests/assertAccessRequestPermissions.js

```diff
@@ -8,6 +8,10 @@ import { getAdminPanelAllowedEntityIds } from '../utilities';
 
 export const assertAccessRequestPermissions = async (accessPolicy, models, accessRequestId) => {
   const accessRequest = await models.accessRequest.findById(accessRequestId);
+  if (accessRequest === null) {
```

Comment:
```suggestion
  if (!accessRequest) {
```
So long as we don't expect that another falsey value will throw us off (like, we actually want `0` to make it through, for example), just using a `!` is more standard (I'm also wanting to avoid a future regression risk if we decided `findById` should return `undefined` rather than `null` if none was found)